packages/@react-spectrum/s2/src/NumberField.tsx

```diff
@@ -199,9 +199,6 @@ export const NumberField = forwardRef(function NumberField(props: NumberFieldPro
                   {label}
                 </FieldLabel>
                 <FieldGroup
-                  role="presentation"
```

Comment:
I assume this was a mistake since it has the actual role=group on it?

Reply:
yeah probably copy and pasted from somewhere. RAC sets the correct role already.